src/lib/openapi.ts

```diff
@@ -1,5 +1,55 @@
+import { writeFile } from 'node:fs/promises'
+
+import { openapi } from '@seamapi/types/connect'
+
 export interface Openapi {
   info: {
     title: string
   }
 }
+
+const extractSchemas = (schemas) => {
+  return Object.entries(schemas).map(([schemaName, schema]) => ({
+    name: schemaName,
+    properties: schema.properties ? Object.keys(schema.properties) : [],
```

Comment:
I think we'll need to more than just property names? Like type, description, required props.